admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -78,7 +79,7 @@ module.exports = {
     maintenance: './js/pages/maintenance',
     manufacturer: './js/pages/manufacturer',
     manufacturer_address_form:
-      './js/pages/manufacturer/manufacturer_address_form',
+        './js/pages/manufacturer/manufacturer_address_form',
```

Comment:
```suggestion
      './js/pages/manufacturer/manufacturer_address_form',
```

We have two  spaces for indentation everywhere